R/optimise_sinus.R

```diff
@@ -71,11 +71,12 @@ optimise_sinus_fixedPeriod <- function(df, period_length = 365.25)
     x0 = atan2(beta, alpha)
   )
 
-  is_max <- which.max(fit.lm2$fitted.values)
-  is_min <- which.min(fit.lm2$fitted.values)
+  get_extreme <- function(FUN) {
+    df$date[as.numeric(names(f(fit.lm2$fitted.values)))]
```

Comment:
Whats `f()` good for? Should it be `FUN` instead?

Reply:
Moin, Moinsen. Yes, I already fixed this.